blog/_posts/2020-02-07-delete-clusters-faster-with-kind.md

```diff
@@ -0,0 +1,104 @@
+---
+slug: delete-clusters-faster-with-kind
+date: 2020-02-07
+author: nick
+layout: blog
+title: "Kill Your Kubernetes Clusters Faster with Kind"
+subtitle: "Announcing a Big Improvement in Local Development"
+tags:
+  - tilt
+keywords:
+  - tilt
+  - kind
+  - local development
+---
+
+TK Selena Gomez Joke
+
+At the ClusterAPI meeting in January
+
+Contributors create a Kind cluster and use Tilt for interactive development
+
+That's why we're excited to announce some big changes to make Kind + Tilt faster than ever
+
+Use our setup script:
+
+TK LINK
+
+The script may look simple. 
+
+A lot of work from a lot of different people in the Kubernetes community went into making this possible.
+
+How does it work?
+
+## What is Kind?
+
+Kind runs a Kubernetes cluster in a Docker container 
+
+Kind is an acronym for Kubernetes IN Docker
+
+Started as a project for testing Kubernetes
+
+Features:
+
+- Fast startup time, faster teardown time
+
+- Works well in continuous integration for testing apps
+
+- Can run multiple versions of Kubernetes, even one you built yourself
+
+Hearing about Kind reminds me of the first time I heard about the Go language. 
+
+Bunch of senior engineers at Google in a corner, trying to redesign C++ to optimize for fast compiler times.
+
+Absolutely changes how you use the tool. You start using it for entirely new use-cases.
+
+## What's New?
+
+To make Kind work for local development, you need:
+
+1) Start the cluster
+2) Install your base services
+3) Update them quickly with your changes
+
+Kind already made #1 fast.
+
+Tilt, with live_update, makes #3 fast.
+
+The missing piece was #2. 
```

Comment:
background on the old state of the art and why it sucked

Reply:
maybe even some background on why `kind load` is necessary at all/talk a bit about how kind can't access your local reg.

Reply:
ooh good point, will add